crt_portal/cts_forms/forms.py

```diff
@@ -1113,20 +1121,15 @@ class Meta:
 
         labels = {
             # These are CRT view only
-            'assigned_section': 'View sections',
             'contact_first_name': 'Contact first name',
             'contact_last_name': 'Contact last name',
             'location_city_town': 'Incident city',
             'location_state': 'Incident state',
             'location_name': 'Incident location name',
             'assigned_to': 'Assignee',
             'public_id': 'Complaint ID',
-            'primary_statute': 'Primary classification',
             'violation_summary': 'Personal description',
-            'create_date_start': 'Created Date Start',
-            'create_date_end': 'Created Date End',
             'contact_email': 'Contact email',
-            'correctional_facility_type': 'Prison Type',
         }
```

Comment:
Are these variables you are moving here used somewhere else?  I am not familiar with the logic of this section.

Reply:
It turns out they're not being used at all! There was no point to keeping them around or keeping them up to date. The places that have these labels are all being set somewhere else.